docs/spec/sources.md

```diff
@@ -362,5 +375,38 @@ _Structured Content Mode_ of the HTTP Protocol Binding for CloudEvents. Sources
 SHOULD send events to its
 [Destination](https://pkg.go.dev/github.com/knative/pkg/apis/duck/v1?tab=doc#Destination).
 
+### CloudEvent `knsource` extension
+
+Sources SHOULD produce CloudEvents with the `knsource` CloudEvent extension
+identifying the originating [Source Custom Object](#source-custom-objects). It
+is highly-recommended to produce the `knsource` CloudEvent extension.
+
+The `knsource` value SHOULD be of the form `<name>.<plural>.<group>`, where:
+
+- `name` is the [Source Custom Object](#source-custom-objects) name
+- `plural` is the [Source CRD](#source-crds)
+  [plural](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#create-a-customresourcedefinition)
+- `group` is the [Source CRD](#source-crds)
+  [API group](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#create-a-customresourcedefinition)
+
+Examples:
+
+- `hello.pingsources.sources.knative.dev`
+- `rabbitmq-source.rabbitmq.sources.knative.dev`
```

Comment:
Isn't that `...rabbitmqsources.sources.knative.dev`, instead of `rabbitmq.sources.knative.dev`?

See: https://github.com/knative-sandbox/eventing-rabbitmq/blob/master/config/source/300-rabbitmqsource.yaml#L28

Reply:
Good catch, thanks!